torch/distributed/tensor/_redistribute.py

```diff
@@ -32,6 +32,71 @@
 
 logger = logging.getLogger(__name__)
 
+# Global configuration flag to control the redistribution planning strategy.
+# When True, forces the graph-based algorithm using Dijkstra's shortest path.
+# When False, forces the greedy algorithm for faster planning.
```

Comment:
not quite correct, right?

"When False, prefers the greedy algorithm for faster planning.  Uses the graph-based algorithm only when necessary to support strided-shard redistribution"